REFERENCE.md

```diff
@@ -210,6 +211,14 @@ When clustering is enabled during training, all subsampling procedures operate o
 
 Note that when clusters are provided, standard errors from `average_treatment_effect` and `average_partial_effect` estimation are also cluster-robust. Moreover, if clusters are specified, then each cluster gets equal weight. For example, if there are 10 clusters with 1 unit each and per-cluster ATE = 1, and there are 10 clusters with 19 units each and per-cluster ATE = 0, then the overall ATE is 0.5 (not 0.05).
 
+### Sample-Weighting
+
+When the distribution of data that you observe is not representative of the population you are interested in scientifically, it can be important to adjust for this. We pass `sample.weights` to specify that in our population of interest, we observe Xi with probability proportional to `sample.weights[i]``. By default, these weights are constant, meaning that our population of interest is the population from which X1 ... Xn are sampled. For causal validity, the weights we use should not be confounded with the potential outcomes --- typically this is done by having them be a function of Xi. One common example is inverse probability of complete case weighting to adjust for missing data, which allows us to work only the complete cases [the units with nothing missing] to estimate properties of the full data distribution [all units as if nothing were missing].
+
+When our estimand is a function of x, e.g. `r(x) = E[Y | X=x]` in `regression_forest` or `tau(x)=E[Y(1)-Y(0)| X=x]` in `causal_forest`, passing weights that are a function of x does not change our estimand. It instead prioritizes fit on our population of interest. In `regression_forest`, this means minimizing weighted mean squared error, i.e. mean squared error over the population specified by the sample weights. In `causal_forest`, this means minimizing weighted R-loss (Nie and Wager (2017), Equations 4/5). When our estimand is an average of such a function, as in `average_treatment_effect` and `average_partial_effect`, it does change the estimand. Our estimand will be the average treatment/partial effect over the population specified by our sample weights.
```

Comment:
Could we add a sentence at the start to explain what the purpose of this paragraph is? Maybe "As a clarification, sample weights only sometimes affect the function we are estimating."

Reply:
Pinging on this again.